integration/kubernetes/runtimeclass_workloads/pod-http-liveness.yaml

```diff
@@ -13,9 +13,9 @@ spec:
   runtimeClassName: kata
   containers:
   - name: liveness
-    image: k8s.gcr.io/liveness
+    image: gcr.io/kubernetes-e2e-test-images/agnhost:2.2
```

Comment:
Is the version number suffix really required? Ideally, we'd remove that to avoid having to maintain these files as new images are released.

Reply:
bad news, it's required. 😭
this image `gcr.io/kubernetes-e2e-test-images/agnhost` doesn't have `latest` tag.
```
$ docker pull gcr.io/kubernetes-e2e-test-images/agnhost
Using default tag: latest
Error response from daemon: manifest for gcr.io/kubernetes-e2e-test-images/agnhost:latest not found

```